src/coda/apps/invoices/views/inspect.py

```diff
@@ -214,6 +233,22 @@ def position_viewmodel(position: Position[ItemType], number: int) -> "PositionVi
     )
 
 
+@login_required
+@require_POST
+def pay_invoice(request: HttpRequest, pk: int) -> HttpResponse:
+    invoice = repository.get_by_id(InvoiceId(pk))
+    invoice.pay()
+    repository.update(invoice)
+    response = (
+        "<input disabled type='text' id='id-head-status' hx-swap-oob='true' value='"
+        + str(invoice.status.name)
```

Comment:
If we decide against reloading the entire page, we need to change this bit.
Building up HTML in a string is potentially dangerous, create a template for that and render it